package.json

```diff
@@ -37,7 +37,7 @@
     "underscore.string": "^3.3.6",
     "use-konami": "^1.0.1",
     "usehooks-ts": "^2.9.1",
-    "vite": "^4.4.9",
```

Comment:
it is good to upgrade vite package to 5 version
but this is something to do with different PR (in my opinion)
we better do only changes solves the problem which PR tries to solve (the issue of PR)

Reply:
this is weird, i don't remember touching the vite package version

Reply:
Next time, make sure not to stage everything in your working directory - adding individual files is a good practice :)

Reply:
the `npm ci` command is failing - it means the `package` and `package-lock` files are not synced.
There are two valid solutions:
1. revert `package.json`
2. run `npm install` and commit the changes in the `package-lock.json` file.

Reply:
Thanks to @shootermv now it's all fixed :)